nextflow.config

```diff
@@ -191,17 +187,15 @@ profiles {
             ]
         }
     }
-    test { includeConfig 'conf/test.config' }
-    test_grch38 { includeConfig 'conf/test_grch38.config' }
-    test_peptides { includeConfig 'conf/test_peptides.config' }
-    test_peptides_h2 { includeConfig 'conf/test_peptides_h2.config' }
```

Comment:
Do we still have tests for mouse?

Reply:
Yes, I added the mouse allele test to `test_peptides` to reduce the number of tests a bit